bool start
bool stop
---
bool is_running